.github/workflows/staging.yml

```diff
@@ -1,9 +1,9 @@
 name: Staging
 on:
   push:
-    branches: [develop]
+    branches: [main]
   pull_request:
-    branches: [develop]
+    branches: [main]
```

Comment:
There was miss understanding, clarify again,  remove this and add ci on pull request (no deploy)